bucket/electron-mail.json

```
@@ -0,0 +1,28 @@
+{
+    "version": "5.1.3",
+    "homepage": "https://github.com/vladimiry/ElectronMail",
+    "description": "An Electron-based unofficial desktop client for ProtonMail.",
+    "license": "GPL-3.0",

```

```suggestion
    "license": "GPL-3.0-only",
```

License can only be `GPL-3.0-only` or `GPL-3.0-or-later`, matching the SPDX identifier.